Exercise_9/entry.js

```diff
@@ -0,0 +1,23 @@
+import MyFramework from './src/MyFramework';
+import routes from './routes';
+
+const wd = new MyFramework();
+const APP_NAME = 'app';
+/* global window:true */
+window.onload = () => {
+  wd.createRoot(APP_NAME)
+    .routes(routes);
+};
+
+window.document.querySelectorAll('.sign-in').forEach((button) => {
```

Comment:
Please, move this block and the next one into `window.onload` block to prevent the execution of the logic before the application has been initialized.